frontend/assets/l10n/app_de.json

```diff
@@ -22,6 +22,16 @@
     "sourceCode": "Quellcode der App",
     "title": "Über"
   },
+  "deeplinkActivation" : {
+    "headline": "Karte aktivieren",
+    "description": "Wollen Sie diese Karte aktivieren?",
+    "limitReached": "Ihr Kartenlimit wurde erreicht. Bitte entfernen Sie vorher eine Karte, damit Sie eine neue aktivieren können.",
+    "buttonText": "Karte aktivieren",
+    "invalidCode": "Ihr Aktivierungscode ist ungültig.",
+    "alreadyExists": "Diese Karte wurde bereits auf dem Gerät aktiviert.",
+    "activationSuccessful": "Die Aktivierung war erfolgreich.",
+    "activationInvalid": "Die Aktivierung befindet sich in einem ungültigen Zustand."
+  },
```

Comment:
Probably, we need locale overrides for Nürnberg (or are we only launching this feature for Bayern?).

Reply:
added